tool/site/client-react/tests/components/claim/__snapshots__/createDispenseClaim.spec.ts.snap

```diff
@@ -51,7 +51,7 @@ Object {
                 },
                 Object {
                   "url": "numberOfRepeatsAllowed",
-                  "valueInteger": 6,
+                  "valueInteger": 5,
```

Comment:
See above - we need to add 1 to `numberOfRepeatsAllowed` when getting it from the dispenseRequest